conf/modules/e_identification_fr.xml

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="e_identification_fr">
+  <doc>
+    <description>Format and send via UART tracking data according to French regulation.</description>
+    <configure name="E_ID_PORT" value="UARTx" description="UART where the e-id device is connected to (UART1, UART2, etc"/>
+    <configure name="E_ID_BAUD" value="B115200" description="UART baud rate"/>
+  </doc>
+  <header>
+    <file name="e_identification_fr.h"/>
+  </header>
+  <init fun="e_identification_fr_init()"/>
+  <periodic fun="e_identification_fr_periodic()" freq="4" autorun="FALSE"/>
+  <makefile target="ap">
+    <configure name="E_ID_PORT"  case="upper|lower"/>
+    <configure name="E_ID_BAUD" default="B115200"/>
+    <define name="USE_$(E_ID_PORT_UPPER)"/>
```

Comment:
You can also disable the RX of this UART because you don't seem to use it.

Reply:
Ok, did the same as in actuators_sbus.xml